### PR TITLE
Removing devtools to separate repository

### PR DESCRIPTION
**Pull request by robons**
I've updated all other pyproject.toml files to reference the pypi package.

Satisfies issue #454

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -266,7 +248,7 @@ pipeline {
                     echo 'mkdocs stash does not exist'
                 }
 
-                archiveArtifacts artifacts: '**/dist/*.whl, **/docs/_build/html/**/*, **/external-docs/site/**/*', fingerprint: true
+                archiveArtifacts artifacts: '**/dist/*.whl, **/docs/_build/html/**/*, **/external-docs/site/**/*, **/test-results.json, **/tox-test-results-*.json, **/*results*.xml', fingerprint: true
```
Comment (robons) [inline Jenkinsfile:251]: @GDonRanasinghe - I've added the test results files to the list of artifacts we should archive to ensure we can figure out what's gone wrong.

File: Jenkinsfile [hunk 2]
```diff
@@ -12,11 +12,6 @@ pipeline {
                     // Clean up any unwanted files lying about after the previous build.
```
Comment (GDonRanasinghe) [inline Jenkinsfile:12]: The https://github.com/GSS-Cogs/csvcubed-devtools project do not have the tests folder. We should make sure the tests folder is also added to the project structure described in https://app.zenhub.com/workspaces/features-squad-61b72275ac896c0010a1b00b/issues/gss-cogs/csvcubed/454

File: Jenkinsfile [hunk 3]
```diff
@@ -12,11 +12,6 @@ pipeline {
...
                     sh "git clean -fxd --exclude='.venv'"
```
Comment (GDonRanasinghe) [inline Jenkinsfile:13]: The tests stages (i.e. pytest and behave) are missing in the Jenkins file at https://app.zenhub.com/workspaces/features-squad-61b72275ac896c0010a1b00b/issues/gss-cogs/csvcubed/454

**Review by GDonRanasinghe — COMMENTED** [review]
@robons Apart from the my two comments around tests, other aspects look good to me.

File: Jenkinsfile (hunk 3 above)
Comment (GDonRanasinghe) [reply Jenkinsfile:13]: Also, there should be a Tox stage within the devtools project's Jenkins file isnt it?

File: Jenkinsfile (hunk 1 above)
Comment (GDonRanasinghe) [reply Jenkinsfile:251]: @robons Many thanks, this will be highly useful.

**Comment by robons** [comment]
Just adding for posterity: 

We don't need tox/unit/behaviour testing infrastructure in this project at the moment given we don't currently have any unit/behaviour tests - see #216.

If/when we add tests in that ticket, we will have to stick testing infrastructure in here. But let's not do the work until we need to.

**Pull request merged**